internal/container/controller.go

```diff
@@ -203,6 +224,84 @@ func (c Controller) ContainerFiles(id string, filespec string) (files []string)
 	return strings.Split(string(stdout), "\n")
 }
 
+func (c Controller) DownloadFile(id string, src string, dest string) {
+	if id == "" {
+		panic("Must pass in non-empty id")
+	}
+	if src == "" {
+		panic("Must pass in non-empty src")
+	}
+	if dest == "" {
+		panic("Must pass in non-empty dest")
+	}
+
+	cmd := []string{"mkdir", "/var/opt/sql/backup"}
+	c.runCmdInContainer(id, cmd)
+
+	_, file := filepath.Split(src)
+
+	// Wget the .bak file from the http src, and place it in /var/opt/sql/backup
+	cmd = []string{
+		"wget",
+		"-O",
+		"/var/opt/sql/backup/" + file, // not using filepath.Join here, this is in the *nix container. always /
```

Comment:
>"/var/opt/sql/backup/" [](http://example.com/codeflow?start=2&length=22)

dest

Reply:
/var/opt/sq/backup is the normal place to place .bak files, used in blog posts etc.

Reply:
sure, but this function has a "dest" parameter. Shouldn't the caller be specifying the var location?